modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -437,16 +437,55 @@ private JSONArray _getFiltersJSONArray(
 				String type = MapUtil.getString(properties, "type");
 
 				if (Objects.equals(type, "date")) {
+					boolean hasPreloadedData;
+
+					String from = MapUtil.getString(properties, "from");
+					String to = MapUtil.getString(properties, "to");
+
+					if (Validator.isNotNull(to) || Validator.isNotNull(from)) {
+						hasPreloadedData = true;
+					}
+					else {
+						hasPreloadedData = false;
+					}
+
 					return JSONUtil.put(
+						"active", hasPreloadedData
+					).put(
 						"entityFieldType", FDSEntityFieldTypes.DATE
 					).put(
 						"id", properties.get("fieldName")
 					).put(
 						"label", properties.get("name")
 					).put(
-						"max", _getDateJSONObject(properties.get("to"))
-					).put(
-						"min", _getDateJSONObject(properties.get("from"))
```

Comment:
@dsanz `min` and `max` are actually missing from UI. Should we add them in Q4?

Reply:
I think so, it looks as a nice-to-have. I've added https://liferay.atlassian.net/browse/LPS-197287 for further refinement @pablo-agulla